app/Cart.php

```diff
@@ -70,10 +70,10 @@ public function removeItem($item)
     public function updateQuantity($item, $qty)
     {
         // Count the number of $items in array
-        $filteredItems = array_filter($this->items, function($value) use ($item) { return $value === $item; });
```

Comment:
My understanding for `array_filter()` filters only the item stated in the callback into a new, filtered array. If you want to keep 3 items, all other items matching that will be toss.

To get the test to pass:
- Set array as param in the new instantiated class
- Set item you want to filter as param in `$this->updateQuantity`
- Set the preferred number of value in the array to `assertEquals` method with the updated quantity of the item


1.